KicadModTree/Point.py

```diff
@@ -136,6 +137,9 @@ def __repr__(self):
     def __str__(self):
         return "(x={x}, y={y})".format(**self.__dict__())
 
+    def distanceTo(self, other):
```

Comment:
Move up to ```round_to``` function, rename to ```distance_to``` for name concistency and use ```Point2D.__arithmetic_parse``` to get the value for ```other```

Reply:
And add a docstring